### PR TITLE
French Translation

### PR DESCRIPTION
**Pull request by thecatcore**

### CODE REVIEW

**Comment by thecatcore** [comment]
@LambdAurora can you review this?

**Comment by Gegy** [comment]
Thanks for the PR! Just added some new translation strings regarding scheduled server shutdown- would you be able to include these too? :)

File: src/main/resources/data/nucleoid_extras/lang/fr_fr.json
Comment (Redblueflame) [inline src/main/resources/data/nucleoid_extras/lang/fr_fr.json]:
```suggestion
	"nucleoid.discord": "nucleoid.xyz/discord",
	"nucleoid.stop.scheduled": "Le redémarrage du serveur a été programmé ! Le serveur va redémarrer dans %s minutes, ou dès que toutes les parties seront terminées.",
        "nucleoid.stop.scheduled.already": "Un redémarrage du serveur a déja été programmé.",
        "nucleoid.stop.game.open": "Impossible de rejoindre cette partie: un redémarrage a été programmé !"
```

**Review by Redblueflame — CHANGES_REQUESTED** [review]
Looks good to me. @Gegy for the 3 missing strings, sent a suggestion with the new translations

File: src/main/resources/data/nucleoid_extras/lang/fr_fr.json
Comment (thecatcore) [reply src/main/resources/data/nucleoid_extras/lang/fr_fr.json]: :thinking: I would rather say `Un redémarrage` in both cases but I need Lambda opinion on this.
We (Lambda and I) have already used `jeux` to translate `games` in plasmid, so I think we should do the same here.

Comment (Redblueflame) [reply src/main/resources/data/nucleoid_extras/lang/fr_fr.json]: I didn't make other translations, so you'll be better than me for that, but there can't be multiple reboot, so Le or Un could potentially be used interchangeably

**Comment by thecatcore** [comment]
Updated this pull request, it just needs to be review by Redblueflame or LambdAurora (or someone who knows french) before being merged.

**Review by Genau6502 — APPROVED** [review]
I'm not fluent, but from my knowledge these translations make sense

**Review by LambdAurora — APPROVED** [review]
imo, the nucleoid discord link shouldn't be a translation string
(nor be translated if it's a literal and it's the same in English)

aside from that, all good

**Pull request merged**
